From 952dc7d427c585bfa551b8b530c5032255a8f3f2 Mon Sep 17 00:00:00 2001
From: Rodrigo Kochenburger <divoxx@gmail.com>
Date: Tue, 13 May 2008 22:08:31 -0300
Subject: [PATCH] Fixing test for named_scope through a has_many :through

The test that checks if a has_many :through association has access to named_scope had a typo and
didn't have the proper fixtures loaded.
---
 activerecord/test/cases/named_scope_test.rb |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/activerecord/test/cases/named_scope_test.rb b/activerecord/test/cases/named_scope_test.rb
index 30c074c..9730f93 100644
--- a/activerecord/test/cases/named_scope_test.rb
+++ b/activerecord/test/cases/named_scope_test.rb
@@ -6,7 +6,7 @@ require 'models/reply'
 require 'models/author'
 
 class NamedScopeTest < ActiveRecord::TestCase
-  fixtures :posts, :authors, :topics
+  fixtures :posts, :authors, :topics, :comments
 
   def test_implements_enumerable
     assert !Topic.find(:all).empty?
@@ -95,7 +95,7 @@ class NamedScopeTest < ActiveRecord::TestCase
   end
 
   def test_has_many_through_associations_have_access_to_named_scopes
-    assert_not_equal Comment.containing_the_letter_e, authors(:david).posts
+    assert_not_equal Comment.containing_the_letter_e, authors(:david).comments
     assert !Comment.containing_the_letter_e.empty?
 
     assert_equal authors(:david).comments & Comment.containing_the_letter_e, authors(:david).comments.containing_the_letter_e
-- 
1.5.4.5

